lib/elixir/test/elixir/application_test.exs

```diff
@@ -19,6 +19,15 @@ defmodule ApplicationTest do
 
     assert Application.delete_env(:elixir, :unknown) == :ok
     assert Application.get_env(:elixir, :unknown, :default) == :default
+
+  end
+
+  test "application information" do
+    assert is_list(Application.started_applications)
```

Comment:
You should check it is a list and then use `List.keyfind/3` to get Elixir out of it and assert on the returned values.